lib/spack/spack/test/url_fetch.py

```diff
@@ -159,6 +163,8 @@ def test_fetch(
             assert 'echo Building...' in contents
 
 
+@pytest.mark.skipif(sys.platform == 'win32',
+                    reason="Not supported on Windows (yet)")
```

Comment:
I think skipping these tests is fine for this PR, although I think restoring list URL functionality should be on the higher end of priority for a follow up PR - so could you add a `TODO-27021` comment to these?

Reply:
Addressed.